manifests/init.pp

```diff
@@ -17,6 +17,7 @@
   $cleanup_on_success   = true,
   $add_build_tools_repo = false,
   $mock_version         = installed,
+  $mirror_url           = undef,
```

Comment:
d'oh! My mind farted on the fact that this is puppet module repo, so of course it's set at the point where it's being used in downstream modules repos.